app/modules/projects/resources.py

```diff
@@ -56,6 +70,11 @@ def post(self, args):
         with context:
             project = Project(**args)
             db.session.add(project)
+        # @todo need to find some way of adding a user to a project.
+        # This way does work, but caused database badness on teardown so temporarity commenting
```

Comment:
curious if this might be related to some `context` closing thing i saw a while ago with some file-reads inside a context.  not a lot of insight, but my guess is @bluemellophone will have some wisdom.

Reply:
Yeah it was a failure to find the user on closing of database. I decided to leave it out and follow up later